### PR TITLE
add goatcounter tracking for support

### PR DESCRIPTION
**Pull request by HoloPollock**
This may not actually work as vue may strip the tags its hard to tell if they are supposed to stay or not

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <template>
 	<footer>
-		<a href="https://ko-fi.com/goodbyte" class="support">Support this site</a>
+		<a
+			data-goatcounter-click="support-us"
+			data-goatcounter-title="Support US"
```
Comment (qjack001) [inline src/components/Footer.vue:5]: why all caps "US"?

Comment (HoloPollock) [reply src/components/Footer.vue:5]: whoops

**Comment by HoloPollock** [comment]
@qjack001 do you know how to get vue to not remove data-* things it seems to they are not in the final html

File: src/components/Footer.vue (hunk 1 above)
Comment (HoloPollock) [reply src/components/Footer.vue]: fixed

**Pull request merged**